app/controllers/concerns/account_brexit_checker_concern.rb

```diff
@@ -5,6 +5,8 @@ module AccountBrexitCheckerConcern
 
   ACCOUNT_ACTIONS = %i[save_results save_results_sign_up save_results_confirm save_results_email_signup save_results_apply saved_results edit_saved_results].freeze
 
+  ATTRIBUTE_NAME = "transition_checker_state"
```

Comment:
This is interesting, why put this in a constant if it's just a string?
Are you thinking this is a concern we might copy paste around, or is this us shooting for better labelling in the code?

Reply:
It was to avoid specifying it twice... though I've just noticed I only actually used the constant in one of the places.